### DISCUSSION TITLE
What does "charged as" mean on the org pricing page?

### DISCUSSION THREAD

**Discussion by Sleavely** [community/community#52929] [New to GitHub] [answered]
I'm viewing the pricing for the organization plans and I'm struggling to understand why there are two vastly different prices per user for each plan. Surely, if the monthly fee is "charged as" $21 then the price is clearly $21?

![](https://i.imgur.com/64NTW3y.png)

**Comment by queenofcorgis** [comment]
Hi there - I reached out to our team about this and we're working on investigating this! Thanks for reporting - once I know more I'll circle back with an update 🌀

**Comment by queenofcorgis** [comment, marked as answer]
Hi there - @Sleavely thanks again for reporting this - it was a bug 🐛 🐞 ! We shipped a fix for this - let me know if you're still encountering any issues.
